20240329_CHWIDE_component_separated/README.md

```diff
@@ -0,0 +1,28 @@
+# Documentation for CHWIDE component separated maps
+
+#### Data description
```

Comment:
What input data did you use? Please provide link

Reply:
Added separate section that describes how the mock simulations are prepared.